discord_thread_summarizer_using_openai/discord_utils.bal

```diff
@@ -0,0 +1,38 @@
+import ballerina/http;
+import ballerina/time;
+
+const DISCORD_API_URL = "https://discord.com/api/v9";
+const time:Seconds DISCORD_EPOCH = 1420070400000.00;
+map<string|string> AUTH_HEADER = {
```

Comment:
```suggestion
map<string> AUTH_HEADER = {
```

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/69/commits/ca9ecd0b4475145df3519080000a28a26d049d7d